conf/routes

```diff
@@ -16,6 +16,8 @@ GET /                                               controllers.Application.inde
 
 GET  /monthly-contributions                         controllers.Application.contributionsRedirect()
 
+GET /geo-redirect                                     controllers.Application.geoRedirect()
```

Comment:
This behaviour would make sense for `/`, is there any reason to create a new endpoint?

Reply:
I'm not sure it does make sense for "/", this is a pretty specific situation where we are diverting people away from `support` to membership, because we want the ability to create a single link that sends people to a different site depending on their location. Or maybe it would make sense but it's not the agreed functionality of the site yet.